### PR TITLE
Example

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Not up to date with last example_ynh*

## Solution

- *Apply last example_ynh*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,27 +1,29 @@
 #sub_path_only rewrite ^__PATH__$ __PATH__/ permanent;
 location __PATH__/ {
-	alias __FINALPATH__/;
 
-	if ($scheme = http) {
-		rewrite ^ https://$server_name$request_uri? permanent;
-	}
+  # Path to source
+  alias __FINALPATH__/ ;
```
Comment (alexAubin) [inline conf/nginx.conf]: I'm thinking that this alias is in fact probably not relevant considerng there's a proxy_pass on that entire location

**Review by alexAubin — APPROVED** [review]

File: conf/nginx.conf (hunk 1 above)
Comment (yalh76) [reply conf/nginx.conf]: I agree, removed

**Comment by yalh76** [comment]
!testme

**Pull request merged**